source/particle/world.cc

```diff
@@ -1279,10 +1281,14 @@ namespace aspect
           prm.declare_entry ("Tracer weight", "10",
                              Patterns::Integer (0),
                              "Weight that is associated with the computational load of "
-                             "a single particle. The sum of tracer weights will be added "
+                             "a single particle. The sum of particle weights will be added "
                              "to the sum of cell weights to determine the partitioning of "
-                             "the mesh. Every cell without tracers is associated with a "
-                             "weight of 1000.");
+                             "the mesh if the 'repartition' particle load balancing strategy "
+                             "is selected. The optimal weight depends on the used "
+                             "integrator and particle properties. In general for a more "
+                             "expensive integrator and more expensive properties a larger "
+                             "particle weight is recommended. Every cell without particles "
+                             "is associated with a weight of 1000.");
```

Comment:
Maybe rephrase this as "Before adding the weights of particles, each cell already carries a weight of 1000 to account for the cost of field-based computations."